lld/test/ELF/Inputs/aarch64-func2-gcs.s

```diff
@@ -0,0 +1,19 @@
+.section ".note.gnu.property", "a"
```

Comment:
`lld/test/ELF/Inputs/aarch64-func2-gcs.s` and `aarch64-func3-gcs.s` are extra files used only once.

We prefer split-file for newer tests (`# RUN: rm -rf %t && split-file %s %t && cd %t` in aarch64-feature-pauth.s)